### DISCUSSION TITLE
Alternative download location

### DISCUSSION THREAD

**Discussion by Vort** [freeplane/freeplane#1388] [Help]
Hello.

Today I decided to try this software.
But surprisingly failed on usually the simplest step - on download process.
Initially, SourceForge just redirected me without allowing to download file.
When I tried to log in there, message about "static offline mode" was shown to me.
Then I tried to look if Archive.org saved binary -> failure again.
Looks like SF prevents archiving of binaries somehow.

Maybe you can upload files also somewhere else?
Like to Releases section of GitHub for example.

**Comment by kryoqa** [comment]
Please provide binaries on github as well. I'm really confused as to why they are **only** available on SourceForge ?

**Comment by walmar** [comment]
There is a related topic on the SourceForge page: [Mirrors - Documentation - SourceForge Support - SourceForge.net](https://sourceforge.net/p/forge/documentation/Mirrors/)
